prompt_toolkit/key_binding/bindings/auto_suggest.py

```diff
@@ -30,7 +30,7 @@ def load_auto_suggest_bindings() -> KeyBindings:
     @Condition
     def suggestion_available() -> bool:
         app = get_app()
-        return (app.current_buffer.suggestion is not None and
+        return (app.current_buffer.suggestion and
```

Comment:
I would probably prefer here to check against ``len(suggestion.text)``, instead of adding the ``__len__`` above. I think it's not semantically correct for ``Suggestion`` to have a length, it's not a container type.

Reply:
@jonathanslenders Thanks. I thought so. I'll try it.